docker/dev.yml

```diff
@@ -23,4 +23,7 @@ x-robotoff-dev: &robotoff-dev
 services:
   api: *robotoff-dev
   workers: *robotoff-dev
-  scheduler: *robotoff-dev
\ No newline at end of file
+  scheduler: *robotoff-dev
+  postgres:
+    ports:
+      - 15432:5432
```

Comment:
We might want to remove this, or does anyone think it might be useful in the future?

Reply:
Since we can access the db through docker, I'm not sure if we also want to expose the ports

Reply:
Alright, I'll remove it.

Reply:
+1